src/components/NostoProduct.ts

```diff
@@ -113,3 +113,10 @@ function registerATCButtons(element: NostoProduct, { addToCart, selectSkuId }: S
     })
   })
 }
+
+function registerSkuData(element: NostoProduct, { setSkus }: Store) {
+  const dataEl = element.querySelector("script[n-sku-data]")
+  if (dataEl) {
```

Comment:
If we rely heavily on this data to be available, should we rather make it mandatory? Otherwise, we don't be able to perform any SKU related functions

Reply:
this data is optional, it's only for synking sku metadata like images and prices to the top level data
the basic ATC functionality works without this

Reply:
Okay. How does it impact image display? Do we fallback to product image when SKU images are not available? Or no images will be displayed?

Reply:
And for the price, do we fallback to top level product price when SKU price is not displayed? 
Do we have tests covering these cases already?

Reply:
I can add tests for prices
for SKU and SKU group selection images and prices on product level are updated if they resolve to a single truthy value

Reply:
added tests